src/main/kotlin/no/nav/bidrag/arbeidsflyt/hendelse/JournalpostHendelseListener.kt

```diff
@@ -1,16 +1,16 @@
 package no.nav.bidrag.arbeidsflyt.hendelse
 
-import no.nav.bidrag.arbeidsflyt.service.HendelseService
+import no.nav.bidrag.arbeidsflyt.service.JsonMapperService
 import org.springframework.kafka.annotation.KafkaListener
 
 interface JournalpostHendelseListener {
     fun lesHendelse(hendelse: String)
 }
 
-class DefaultJournalpostHendelseListener(private val hendelseService: HendelseService) : JournalpostHendelseListener {
+class DefaultJournalpostHendelseListener(private val jsonMapperService: JsonMapperService) : JournalpostHendelseListener {
 
     @KafkaListener(groupId = "bidrag-arbeidsflyt", topics = ["\${TOPIC_JOURNALPOST}"], errorHandler = "hendelseErrorHandler")
     override fun lesHendelse(hendelse: String) {
-        hendelseService.lesHendelse(hendelse)
+        jsonMapperService.lesHendelse(hendelse)
```

Comment:
endrer hvordan service fungerer... mapper returnerer et objekt som brukes i behandling... to be continued...